contracts/payloads/IGP112/PayloadIGP112.sol

```diff
@@ -117,65 +125,61 @@ contract PayloadIGP112 is PayloadIGPMain {
-
-        // TYPE_1 vault - pause both supply and borrow
-        // Pause supply limits
-        setSupplyProtocolLimitsPaused(vault_, constants_.supplyToken);
-
-        // Pause borrow limits
-        setBorrowProtocolLimitsPaused(vault_, constants_.borrowToken);
-
-        // Pause user operations
-        address[] memory supplyTokens = new address[](1);
-        supplyTokens[0] = constants_.supplyToken;
-
-        address[] memory borrowTokens = new address[](1);
-        borrowTokens[0] = constants_.borrowToken;
+        VaultWithdrawalLimit[] memory supplyLimits_ = new VaultWithdrawalLimit[](10);
+        supplyLimits_[0] = VaultWithdrawalLimit({vaultId: 1, baseWithdrawalLimitInUSD: 2_200 * 1e2}); // ETH/USDC
+        supplyLimits_[1] = VaultWithdrawalLimit({vaultId: 2, baseWithdrawalLimitInUSD: 3_200 * 1e2}); // ETH/USDT
+        supplyLimits_[2] = VaultWithdrawalLimit({vaultId: 3, baseWithdrawalLimitInUSD: 2_600 * 1e2}); // wstETH/ETH
+        supplyLimits_[3] = VaultWithdrawalLimit({vaultId: 4, baseWithdrawalLimitInUSD: 2_250 * 1e2}); // wstETH/USDC
+        supplyLimits_[4] = VaultWithdrawalLimit({vaultId: 5, baseWithdrawalLimitInUSD: 2_270 * 1e2}); // wstETH/USDT
+        supplyLimits_[5] = VaultWithdrawalLimit({vaultId: 6, baseWithdrawalLimitInUSD: 14_500_000 * 1e2}); // weETH/wstETH
+        supplyLimits_[6] = VaultWithdrawalLimit({vaultId: 7, baseWithdrawalLimitInUSD: 4_000 * 1e2}); // sUSDe/USDC
+        supplyLimits_[7] = VaultWithdrawalLimit({vaultId: 8, baseWithdrawalLimitInUSD: 520 * 1e2}); // sUSDe/USDT
+        supplyLimits_[8] = VaultWithdrawalLimit({vaultId: 9, baseWithdrawalLimitInUSD: 3_450_000 * 1e2}); // weETH/USDC
+        supplyLimits_[9] = VaultWithdrawalLimit({vaultId: 10, baseWithdrawalLimitInUSD: 1_750_000 * 1e2}); // weETH/USDT
+
+        for (uint256 i = 0; i < supplyLimits_.length; i++) {
+            address vault_ = getVaultAddress(supplyLimits_[i].vaultId);
+            IFluidVaultT1.ConstantViews memory constants_ = IFluidVaultT1(vault_).constantsView();
+
+            SupplyProtocolConfig memory supplyConfig_ = SupplyProtocolConfig({
+                protocol: vault_,
+                supplyToken: constants_.supplyToken,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimitInUSD: supplyLimits_[i].baseWithdrawalLimitInUSD
+            });
+            setSupplyProtocolLimits(supplyConfig_);
+        }
 
-        LIQUIDITY.pauseUser(vault_, supplyTokens, borrowTokens);
+        // Apply "paused" borrow limits to all vaults 1-10 (without actually pausing)
+        for (uint256 vaultId = 1; vaultId <= 10; vaultId++) {
+            address vault_ = getVaultAddress(vaultId);
+            IFluidVaultT1.ConstantViews memory constants_ = IFluidVaultT1(vault_).constantsView();
+
+            BorrowProtocolConfig memory borrowConfig_ = BorrowProtocolConfig({
```

Comment:
can use the existing helper method from helpers.sol in common folder  `setBorrowProtocolLimitsPaused`